chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md

```diff
@@ -576,13 +576,13 @@ $$
 \beta_1 \\
 \vdots \\
 \beta_n
-\end{bmatrix} = \boldsymbol{\beta}^\top.
+\end{bmatrix} = \boldsymbol{\beta}.
 $$
 
 This illustrates a few factors about matrix calculus that we will often counter throughout this section:
 
 * First, The computations will get rather involved.
-* Second, The final results are much cleaner than the intermediate process, and will always look similar to the single variable case.  In this case, note that $\frac{d}{dx}(bx) = b$ and $\frac{d}{d\mathbf{x}} (\boldsymbol{\beta}\mathbf{x}) = \boldsymbol{\beta}^\top$ are both similar. 
+* Second, The final results are much cleaner than the intermediate process, and will always look similar to the single variable case.  In this case, note that $\frac{d}{dx}(bx) = b$ and $\frac{d}{d\mathbf{x}} (\boldsymbol{\beta}^\top\mathbf{x}) = \boldsymbol{\beta}$ are both similar. 
```

Comment:
I know it might be tricky here that the \beta is a **row** vector, rather than conventionally a **column** vector. 

I like this updated version. So can you add one more modification:
By changing "Suppose that we have some fixed row vector  𝜷" to "Suppose that we have some fixed column vector  𝜷"?

Reply:
I have changed it in commit 6ef2874021856ec2cbfe2cecc6c14a4a619a0297.
https://github.com/d2l-ai/d2l-en/pull/1112/commits/6ef2874021856ec2cbfe2cecc6c14a4a619a0297

Reply:
I have changed it in 6ef2874021856ec2cbfe2cecc6c14a4a619a0297.